src/modules/Builder/components/Configuration/constants/configuration.js

```diff
@@ -0,0 +1,11 @@
+export const EXECUTE_STAGE = "BUILDER/CONFIGURATION/EXECUTE_STAGE";
```

Comment:
Use one style of writing constants (through an empty line), as in `src/modules/Builder/components/Configuration/actions/configuration.js`.